### ISSUE TITLE
Issue displaying negative zero fractions

### ISSUE THREAD

**Issue by bmatthews** [a14n/dart-decimal#65]
Reproducing test this returns 0.0002 not -0.0002

```dart
test('toString', () {
    expectThat(dec('-0.0002').toString()).equals('-0.0002');
  });
```

I assume this is because of this function

```dart
  /// A decimal-point string-representation of this number with [fractionDigits]
  /// digits after the decimal point.
  String toStringAsFixed(int fractionDigits) {
    assert(fractionDigits >= 0);
    if (fractionDigits == 0) return round().toBigInt().toString();
    final value = round(scale: fractionDigits);
    final intPart = value.toBigInt();
    final decimalPart =
        (one + value.abs() - intPart.abs().toDecimal()).shift(fractionDigits);
    return '$intPart.${decimalPart.toString().substring(1)}';
  }
```

as `value.toBigInt()` probably does not return a negative number for `-0`

**Comment by bmatthews** [comment]
added a new test and altered the code to resolve in https://github.com/a14n/dart-decimal/pull/66

**Comment by a14n** [comment]
fixed in decimal-2.0.1
